labs/lfdt/hyperledger_fabric_flutter_labs.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: hyperledger_fabric_flutter_labs
+parent: LFDT Labs
+grand-parent: Active Labs
+SPDX-License-Identifier: CC-BY-4.0
+---
+
+# Lab Name
+
+Hyperledger Fabric Flutter Labs .
+
+# Short Description
+
+Experimental playground for Hyperledger Fabric on Flutter and Dart
+platforms .
+
+# Scope of Lab
+
+Extend original fabric_client_flutter lab filed within Hyperledger Labs
+.
+
+Filed: ../hyperledger/fabric_client_flutter.md .
+
+Hyperledger Labs homepage:
+https://lf-decentralized-trust-labs.github.io/labs/hyperledger/fabric_client_flutter.html
+.
+
+The extension should allow purely client side funtionality on Flutter
+and Dart platforms through Java interop package : jnigen (
+https://pub.dev/packages/jnigen ) bindings of Hyperledger Fabric Gateway
+Client API for Java in support of Hyperledger Fabric v2.4 + .
+
+The functionalities include but not limited to client offline signing ,
+offline TLS CA certificate loading on mobile platforms ( e.g. Android )
+without intermidiate parties, ACME servers or CSR ( Certificate Signing
+Request ) , which eliminates the requirements of [ FABN-895 ] (
+https://docs.google.com/document/d/1gj5XB7yS-pfjpvZEUQh5lBGSIE6aQemu8A69tAYQtTc
+) .
+
+This lab also deduplicates or eliminates the need of a server or
+fabric-server-node ( ../archived/fabric-server-node.md ) .
+
+The lab introduces a playground for high-level gateway APIs for
+Hyperledger Fabric version 2.4 or later on Flutter and Dart platforms .
```

Comment:
Has the project team considered extending previous lab to support newer versions of Fabric? I'm trying to understand the reason for a new project.

Reply:
yes . the reason for a new lab can be seen on the section # rationale
and alternatives (
https://github.com/hyperledger/fabric-rfcs/pull/67/files#diff-9a02df37701ce2c2cc26070f36b7a88b25c1bfe4a6c8292f76045ec32297c75cR101-R117
) of the rfc of this lab .